src/impl/index.ts

```diff
@@ -224,11 +249,14 @@ export default class OpenTelemetryTracingImpl {
   };
 
   private getInstrumentationPlugins = () => {
-    const { plugins, corsUrls } = this.props;
+    const { plugins, corsUrls, plugin_config } = this.props;
     const insrumentations: any = [];
 
     // XMLHttpRequest Instrumentation for web plugin
-    if (plugins.instrument_xhr !== false) {
+    if (plugin_config.instrument_xhr && plugin_config.instrument_xhr.enabled !== false) {
```

Comment:
would `plugin_config.instrument_xhr?.enabled !== false` also work?

Reply:
fixed it